source/connectors/kafka-sink-data-formats.txt

```diff
@@ -9,10 +9,338 @@ Kafka Sink Connector Data Formats
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-Sink Data Formats
------------------
 
-Lorem ipsum
+Supported Data Formats
+----------------------
+
+The MongoDB Kafka Sink Connector ``converter`` setting specifies the
+deserialization method for data it reads from a topic. The converter
+can deserialize the following data formats:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 1 4
+
+   * - Format Name
+     - Description
+
+   * - Avro
```

Comment:
Should this be AVRO to match the other mentions of it?

Reply:
Good catch, will fix.